src/xrGame/saved_game_wrapper.cpp

```diff
@@ -161,10 +163,15 @@ CSavedGameWrapper::CSavedGameWrapper(LPCSTR saved_game_name)
         {
             CGameGraph graph(*chunk);
             m_level_id = graph.vertex(object->m_tGraphID)->level_id();
+
             if (graph.header().level_exist(m_level_id))
                 m_level_name = graph.header().level(m_level_id).name();
+                
```

Comment:
Maybe it would be better to add brackets instead of adding additional line for improving readability?

Reply:
It's not clear for me how we can add brackets. Like this:
```
{
if (…)

}
else (…)
```
… or like this …
```
{
if (….)
else (….){}
}
```
?